coalib/parsing/filters/CanDetectFilter.py

```diff
@@ -0,0 +1,21 @@
+from coalib.parsing.filters.FilterBase import FilterBase
+
+
+class CanDetectFilter(FilterBase):
+
+    def filter_section_bears(self, bears, args):
```

Comment:
Exactly. I had thought of making classmethod/staticmethod, but it didn't work with the obvious reason you specified.